govcd/org.go

```diff
@@ -23,6 +26,23 @@ func NewOrg(c *Client) *Org {
 	}
 }
 
+// AdminOrg gives an admin representation of an org
+// Users can delete, update orgs with an admin org object
+// AdminOrg users have to get an org representation to use find Catalogs
+type AdminOrg struct {
+	AdminOrg *types.AdminOrg
+	Org
+}
+
+func NewAdminOrg(c *Client) *AdminOrg {
+	return &AdminOrg{
+		AdminOrg: new(types.AdminOrg),
+		Org: Org{
+			c: c,
```

Comment:
We really must clean up the library of this kind of one-letter variables, fields, and types.